frontend/src/components/MultiSelect.js

```diff
@@ -248,6 +248,7 @@ MultiSelect.propTypes = {
     hideSelectedOptions: PropTypes.bool,
   }),
   disabled: PropTypes.bool,
+  /** must be an object with a function called 'validate'. */
   rules: PropTypes.shape({}),
```

Comment:
```suggestion
  rules: PropTypes.shape({
    validate: PropTypes.function
  }),
```

??

Reply:
This was just to clarify what needs to happen in order to add a custom validate.